pkgs/tools/security/arti/default.nix

```diff
@@ -10,18 +10,18 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "arti";
-  version = "1.2.6";
+  version = "1.2.7";
```

Comment:
While building locally a test seems to be hung:
`building arti-1.2.7 (checkPhase): test reload_cfg::test::watch_multiple has been running for over 60 seconds`

Reply:
This looks to be the cause of a test that was enabled right before the release. Gave a try to patch this, but for now, the second patch is still failing:

```
  patches = [
    # This makes the event debouncing optional, and disables it in the tests
    (fetchpatch {
      url = "https://gitlab.torproject.org/tpo/core/arti/-/merge_requests/2387.patch";
      hash = "sha256-733eH9t6BBp3kjFbvJZvS4CElM0ds3s3i1sMPwJSl5g=";
    })
    # Temporarily disable a flaky test
    (fetchpatch {
      url = "https://gitlab.torproject.org/tpo/core/arti/-/merge_requests/2391.patch";
      hash = "sha256-ecJltiZGoYv73iemxXI2N8pT83zSZXVjHJJpzk2fD7U=";
    })
  ];
```

Reply:
Probably the best approach for this release would be to just disable both test cases since these are both reworked/fixed after the release.